ops/services/alerts/app_service_metrics/exceptions.tf

```diff
@@ -0,0 +1,53 @@
+# To do log-based alerts, we need to make sure we're actually collecting AppServiceConsoleLogs from the API
+data "azurerm_resource_group" "management" {
+  name = "prime-simple-report-management"
+}
+
+data "azurerm_log_analytics_workspace" "global" {
+  name                = "simple-report-log-workspace-global"
+  resource_group_name = data.azurerm_resource_group.management.name
+}
+
+resource "azurerm_monitor_diagnostic_setting" "collect_appserviceconsolelogs" {
+  name                       = "API App Service console logs"
+  target_resource_id         = var.app_service_id
+  log_analytics_workspace_id = data.azurerm_log_analytics_workspace.global.id
+
+  log {
+    category = "AppServiceConsoleLogs"
+    enabled  = true
+
+    retention_policy {
+      days    = 7
+      enabled = true
+    }
+  }
+}
+
+# Add an alert for GraphQL query validation failures (more than 3 in a 5-minute window)
+resource "azurerm_monitor_scheduled_query_rules_alert" "graphql_query_validation_failures" {
+  name                = "graphql-query-validation-failures"
```

Comment:
Should this have `var.env` in it, or is it sufficiently clearly scoped to some container resource (which one? I was thinking app service plan but that doesn't appear to be one of the arguments) that we don't care about the name collision?

Reply:
I don't actually know, but that's a good catch, and I'm going to parameterize it anyway.